scripts/check-host-zero-nanollm_test.sh

```
@@ -73,6 +73,72 @@ rm -f "${tmp_manifest}"
+# one line per assertion.
+render_path_violation_quiet() { render_path_violation "$1" >/dev/null; }
+
+healthy_deps="github.com/invakid404/baml-rest/internal/nativeprompt
+github.com/invakid404/baml-rest/internal/bamlprofile
+github.com/invakid404/minijinja-go/v2
+github.com/invakid404/minijinja-go/v2/value
+strings"
+
+# Mutations are built with the REAL grep, before the mocked-error block below.
+deps_without_profile="$(grep -Fv 'baml-rest/internal/bamlprofile' <<<"${healthy_deps}")"
+deps_without_fork="$(grep -Fv 'invakid404/minijinja-go' <<<"${healthy_deps}")"
```

Shellcheck (0.11.0)

[error] This function is only defined later. Move the definition up.

(SC2218)